app/sms/server/services/zenvia.js

```diff
@@ -51,7 +51,8 @@ class Zenvia {
 		};
 		returnData.media = [];
 
-		if(data.message.contents.length>0){
+		const { contents = [] } = data.message;
+		if (contents.length > 0) {
```

Comment:
It looks much better now, but you need to the same in the entire code, not just this line.